src/Kernel/client/ExecutionPathVisualizer/styles.ts

```diff
@@ -14,112 +22,138 @@ export interface StyleConfig {
     classicalOne?: string;
 }
 
-const defaultStyleConfig: StyleConfig = {
+const defaultStyle: StyleConfig = {
+    lineStroke: "#000000",
+    lineWidth: 1,
+    textColour: "#000000",
     unitary: "#D9F1FA",
+    oplus: "#FFFFFF",
     measure: "#FFDE86",
     classicalUnknown: "#E5E5E5",
     classicalZero: "#C40000",
     classicalOne: "#4059BD",
 };
 
+const blackAndWhiteStyle: StyleConfig = {
+    lineStroke: "#000000",
+    lineWidth: 1,
+    textColour: "#000000",
+    unitary: "#FFFFFF",
+    oplus: "#FFFFFF",
+    measure: "#FFFFFF",
+    classicalUnknown: "#FFFFFF",
+    classicalZero: "#FFFFFF",
+    classicalOne: "#FFFFFF",
+};
+
+const invertedStyle: StyleConfig = {
+    lineStroke: "#FFFFFF",
+    lineWidth: 1,
+    textColour: "#FFFFFF",
+    unitary: "#000000",
+    oplus: "#000000",
+    measure: "#000000",
+    classicalUnknown: "#000000",
+    classicalZero: "#000000",
+    classicalOne: "#000000",
+};
+
+/**
+ * Set of default styles.
+ */
+export const STYLES: { [name: string]: StyleConfig } = {
```

Comment:
Does the all caps indicate something special here?

Reply:
It's just a [JavaScript convention](https://www.w3schools.com/js/js_conventions.asp) for naming constants haha

Reply:
Got it. I saw the other `const` things in this file weren't like that, but I guess this one is exported. Just curious, thanks!

Reply:
Oh yea, that's a good catch. Yea, I guess the other `const` variables are locally used whereas this is an exported one to be used "externally". Not sure if it's just my own personal style haha!

Reply:
My general rule of thumb is that if you are a human, personal style is allowed.